tldraw/tldraw
# Fix dropdown menus not opening in focus examples

Rel:  #4253 

Dropdown menus weren't opening in focus examples. By sidestepping blurring and focusing the container we can get them working again. However, this is a band-aid fix until the underlying problem can be solved. Something to do with editor's container getting refocused before the dropdown menu has a chance to open.

### Change type


### Test plan

1. Open the inline example and editor focus example
2. Click to open the style panel, more tools menu, pages menu or main menu
3. Menus should open as expected.

### Release notes

- Fixed a bug with dropdown menus not opening on editor focus examples
